### PR TITLE
enhance(integration-tests): remove deprecated properties usage

### LINKED ISSUE #660: Update deprecated properties in functional tests
In test logs:
```
    2020-01-29 15:53:15.569 ERROR 5559 --- [      events-11] o.s.b.c.p.m.PropertiesMigrationListener  : 
    The use of configuration keys that are no longer supported was found in the environment:
    Property source 'applicationConfig: [classpath:/application-base.yml]':
    	Key: management.metrics.binders.files.enabled
    		Reason: Instead, filter 'process.files' metrics.
    	Key: management.metrics.binders.processor.enabled
    		Reason: Instead, filter 'system.cpu' and 'process.cpu' metrics.
    	Key: management.metrics.binders.uptime.enabled
    		Reason: Instead, filter 'process.uptime' and 'process.start.time' metrics.

    		    2020-01-29 15:53:15.568  WARN 5559 --- [      events-11] o.s.b.c.p.m.PropertiesMigrationListener  : 
    The use of configuration keys that have been renamed was found in the environment:
    Property source 'applicationConfig: [classpath:/application-base.yml]':
    	Key: management.metrics.binders.jvm.enabled
    		Replacement: management.metrics.enable.jvm
    	Key: management.metrics.binders.logback.enabled
    		Replacement: management.metrics.enable.logback
```

### PR DESCRIPTION
**Pull request by Aloren**
fixes #660

### CODE REVIEW

**Comment by Aloren** [comment]
Looks like the build failed due to some reason not related to the pr:
```
> Task :kayenta-mannwhitney:compileKotlin
error: error reading /home/runner/.gradle/caches/modules-2/files-2.1/org.springframework/spring-webmvc/5.2.3.RELEASE/745a62502023d2496b565b7fe102bb1ee229d6b7/spring-webmvc-5.2.3.RELEASE.jar; error in opening zip file

> Task :kayenta-mannwhitney:compileJava
error: error reading /home/runner/.gradle/caches/modules-2/files-2.1/org.springframework/spring-webmvc/5.2.3.RELEASE/745a62502023d2496b565b7fe102bb1ee229d6b7/spring-webmvc-5.2.3.RELEASE.jar; cannot read zip file

> Task :kayenta-core:compileKotlin
w: Configure scripting: unable to process classpath entry /home/runner/.gradle/caches/modules-2/files-2.1/org.springframework/spring-webmvc/5.2.3.RELEASE/745a62502023d2496b565b7fe102bb1ee229d6b7/spring-webmvc-5.2.3.RELEASE.jar: java.util.zip.ZipException: error in opening zip file
```

**Comment by fieldju** [comment]
Looks like the build is failing because the edited yaml is malformed

image

File: kayenta-integration-tests/src/test/resources/application-base.yml [hunk 1]
```diff
@@ -35,21 +35,13 @@ server:
...
-        enabled: false
-      logback:
-        enabled: false
-      jvm:
-        enabled: false
+    exclude: >
+    org.springframework.boot.actuate.autoconfigure.metrics.web.tomcat.TomcatMetricsAutoConfiguration,
+    org.springframework.boot.actuate.autoconfigure.metrics.JvmMetricsAutoConfiguration,
+    org.springframework.boot.actuate.autoconfigure.metrics.LogbackMetricsAutoConfiguration,
+    org.springframework.boot.actuate.autoconfigure.metrics.SystemMetricsAutoConfiguration,
```
Comment (fieldju) [inline kayenta-integration-tests/src/test/resources/application-base.yml]: Lines 39-42 need to be indented one more time

Comment (fieldju) [reply kayenta-integration-tests/src/test/resources/application-base.yml]: http://www.yamllint.com/

Comment (Aloren) [reply kayenta-integration-tests/src/test/resources/application-base.yml]: Thank you! :)

**Pull request merged**